### PR TITLE
Added url replacement for Twitter

### PR DESCRIPTION
**Pull request by kunaldes**
Gets half way to #16. Still need to figure out media.

Figured I'd get your opinion on how I did lines 37/38

Also I don't think there are enough variables/properties called "text". I should add some more.

### CODE REVIEW

File: commands/twitter.js [hunk 1]
```diff
@@ -26,17 +26,21 @@ export class Twitter extends Command {
...
-      twitter.getTweet({id: tweet_id},
+      twitter.getTweet({id: tweet_id, include_entities: true},
         (error) => {
           console.log(`ERROR: Twitter Info - ${error}`);
           reject();
         },
         (success) => {
           let data = JSON.parse(success);
+          let text = data.text;
+          data.entities.urls.reverse().forEach(url => {
```
Comment (IanMitchell) [inline commands/twitter.js]: The reverse ensures that indices stay accurate, right? Since you're doing in-place swapping?

Comment (kunaldes) [inline commands/twitter.js]: Yeah. I don't know of a way to swap all the links at once, so i go from the end of the string to the beginning so the indices in the twitter response stay accurate.

The results from twitter are in the order they appear in the tweet, but I don't think there's actually a guarantee of that. I think it's safe to assume they'll be in order though.

Comment (IanMitchell) [inline commands/twitter.js]: Any reason you're not doing `text = text.replace(url.url, url.expanded_url);`?

Comment (kunaldes) [inline commands/twitter.js]: I found the description "some or all matches of a pattern replaced by a replacement" worrisome and didn't bother testing how it actually worked lol

Comment (IanMitchell) [inline commands/twitter.js]: Right, I remember now

**Pull request closed without merging**